src/_data/capability-matrix.yml

```diff
@@ -668,6 +762,10 @@ categories:
             l1: 'Yes'
             l2: fully supported
             l3: ''
+          - class: mapreduce
+            l1: 'Yes'
```

Comment:
This stuff is another area where it doesn't really make sense for a batch-only runner to have any answer.

Reply:
done